docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1097,17 +1079,17 @@ fail! Here's a few:
 
 <strong>Core Concept</strong>
 
-With Cypress, you don't have to assert to have a useful test. Even without
-assertions, a few lines of Cypress can ensure thousands of lines of code are
-working properly across the client and server!
+With Cypress, you don't have to write explicit assertions to have a useful test.
+Without a single `expect()` or `.should()`, a few lines of Cypress can ensure
+thousands of lines of code are working properly across the client and server.
```

Comment:
I actually like the explication points in this guide since it's the intro and trying to get you hyped 🙃

Reply:
Hm, I tend to prefer a more steady tone - hype turns me away from docs, and would prefer for features to speak for themselves.